### ISSUE TITLE
Timeline to adopt newer DAPR versions

### ISSUE THREAD

**Issue by hortocam** [microsoft/azure-container-apps#1195]
There are bug fixes in DAPR that are in recent releases from 1.13.0 through 1.13.4.

The FAQ suggests that implementation of new versions follows an approximate 6 week delay.  It seems like we are well beyond that for 1.13.0.  

Could you please provide some insight on when we might see this update?

**Comment by JasonRodman** [comment]
Any news on this? We have an issue with a dapr component that has a fix in 1.13 that we are waiting on that is causing daily issues for us.

**Comment by greenie-msft** [comment]
Hi @JasonRodman and @hortocam 

We're actively working on stabilizing Dapr 1.13 in ACA. Current ETA  is end of August. 

What component bug are you looking for?

**Comment by JasonRodman** [comment]
Thats good news. The error we are experiencing was reported here https://github.com/dapr/components-contrib/issues/3325 and the fix was cherry picked into 1.13.

**Comment by jacqueskang** [comment]
When is it planned for dapr 1.14 please? We are interested by using managed identity (Entra ID) for authentication against Azure Redis Cache

https://docs.dapr.io/reference/components-reference/supported-state-stores/setup-redis/

**Comment by JasonRodman** [comment]
> Hi @JasonRodman and @hortocam
> 
> We're actively working on stabilizing Dapr 1.13 in ACA. Current ETA is end of August.
> 
> What component bug are you looking for?

Any news on this? It's the middle of September now and I need to tell my organization when to expect this update.

**Comment by BBTristanBenschop** [comment]
any chance we will also see support for https://blog.dapr.io/posts/2024/03/05/dapr-v1.13-is-now-available/#graceful-shutdown when 1.13 is released?

**Comment by DawnDevelop** [comment]
I feel like this is kind of an urgent issue since version 1.13 also contains a lot of Bug fixes and we are experiencing some of the issues which have been fixed in version 1.13. Is there any update on this?

**Comment by JasonRodman** [comment]
> Looking forward to 1.14 at this point, for the redis managed identity auth feature.

Since I was not getting any answers here, I decided to send a support request. Here was their response:

"I’ve just received an update from our Product Group Team, who are responsible for rolling out Dapr updates. According to the current status, version testing has been completed, and the release process is in its final stages before the rollout begins. Since this is a major version update, the team is taking extra precautions, so we can expect the new version to be available sometime in the first quarter of 2025."

I can only imagine 1.14 would take until end of next year at this rate.

**Comment by BlackRider97** [comment]
> > Looking forward to 1.14 at this point, for the redis managed identity auth feature.
> 
> Since I was not getting any answers here, I decided to send a support request. Here was their response:
> 
> "I’ve just received an update from our Product Group Team, who are responsible for rolling out Dapr updates. According to the current status, version testing has been completed, and the release process is in its final stages before the rollout begins. Since this is a major version update, the team is taking extra precautions, so we can expect the new version to be available sometime in the first quarter of 2025."
> 
> I can only imagine 1.14 would take until end of next year at this rate.

Yes, this accurate for upcoming Dapr 1.13 release.  and you don't have wait for this longer for future releases. We will communicate our plans for Dapr 1.14 once 1.13 is fully rolled out for all customers.

**Comment by BC89** [comment]
Now we are on to Dapr 1.15 and the delays continue.

**Comment by ManuInNZ** [comment]
any comms on progress and next steps for supporting for example a selectable version of Dapr per Apps Environments maybe ?

**Comment by JasonRodman** [comment]
> any comms on progress and next steps for supporting for example a selectable version of Dapr per Apps Environments maybe ?

Microsoft reached out to us since we had an open support ticket. They notified us that the product group had started production deployment of 1.13. They said it's going to take some time to deploy all clusters. They offered to have us be upgraded this week as part of early testing once they have more validation in canary and pilot regions.

**Comment by owns** [comment]
I am looking forward to 1.15 for [pubsub subscription streaming](https://docs.dapr.io/developing-applications/building-blocks/pubsub/subscription-methods/#streaming-subscriptions) 😁
But I'd image that'll be a while since we have no real timeline for 1.14

**Comment by AnthonyFontaine-ES** [comment]
Any hint on when to expect 1.15 as a minimum?  There's an issue where Event Hub messages are not processed sequentially and Dapr 1.15 fixes that, which is a must have if you have IOT curation to process.  

Also, the doc mentions to post questions on Discord, but it doesn't seems there's someone from the team owning the project there.

**Comment by jpiquot** [comment]
> Any hint on when to expect 1.15 as a minimum? There's an issue where Event Hub messages are not processed sequentially and Dapr 1.15 fixes that, which is a must have if you have IOT curation to process.
> 
> Also, the doc mentions to post questions on Discord, but it doesn't seems there's someone from the team owning the project there.

Given that the previous rollouts took 18 months, I don't expect version 1.15 until at least mid-2026.

**Comment by espenairmine** [comment]
Any update on this would be welcome

**Comment by MatteoCalabro-TomTom** [comment]
This outdated version blocks opentelemetry upgrades, due to a schema change in otel 1.37.

**Comment by calleo** [comment]
A very annoying bug that has been fixed in 1.15: https://github.com/dapr/components-contrib/issues/3573

**Comment by Muthukumar123** [comment]
Are there any updates? when version 1.15 in ACA? We need DAPR workflow to be integrated in ACA

**Comment by tse-at-work** [comment]
Is there a status update regarding updating? We want to use DAPR Jobs, this too is released in 1.15.

**Comment by JasonRodman** [comment]
My advice if you have a service agreement is to open a service ticket with them. It's the only way you get any answers these days about this. Took me 6 months last time to get any dialog going with them. It didn't speed anything up, but at least we got some info.

**Comment by Sjark** [comment]
Is there any status update on this? We also really need 1.15 for pubsub subscription streaming.

**Comment by mnoconexus** [comment]
Any info on which version of Dapr you are adopting to?

**Comment by BlackRider97** [comment]
Thank you everyone for your patience and feedback on this thread over the past two years. We hear you — the upgrade cadence has been too slow, and we are committed to improving this.

We're now actively rolling out a **staged Dapr upgrade from v1.14 → v1.15 → v1.16.4**, as announced in [#1676](https://github.com/microsoft/azure-container-apps/issues/1676). This rollout began end of March 2026 and is expected to complete by end of April 2026.

This brings many of the features and fixes requested in this thread including: Dapr Workflows, pubsub streaming subscriptions, EventHub ordering fixes, OTel improvements, and more.

Closing this in favor of [#1676](https://github.com/microsoft/azure-container-apps/issues/1676) where you can track progress. For future Dapr version updates, we will post announcements proactively.

---
*🤖 This comment was generated with AI assistance and may contain inaccuracies. If this doesn't address your concern or if the issue persists, please reopen this issue or create a new one.*

**Comment by stweb1963** [comment]
What is the timeline to release 1.17+. It addresses many workflow issues, etc.
